.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+    "recommendations": [
+        "Dart-Code.dart-code",
+        "Dart-Code.flutter",
+        "eamodio.gitlens",
```

Comment:
My thinking was it would nudge our environments towards the same standard, making collaboration easier, but I will remove all but dart and flutter extensions from here.